src/main/webapp/js/nav.js

```diff
@@ -4,30 +4,107 @@ function redirectToUrl(url) {
     barba.go(url)
 }
 
-const loadingScreen = document.querySelector('.loading-screen')
-const mainNavigation = document.querySelector('.main-navigation')
+function isTransitioning() { return document.querySelector('html').classList.contains('is-transitioning') }
 
-function pageTransitionIn() {
+function capitalizeFirstLetter(string) {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+function defaultTransition(currContainer, nextContainer) {
+    const transitionTitle = document.querySelector('.transition-title');
+    const transitionBackground = document.querySelector('.transition-background');
+    transitionTitle.innerHTML = capitalizeFirstLetter(nextContainer.dataset.barbaNamespace);
```

Comment:
would it be easier to do this in css? `text-transform: capitalize;`

Reply:
I didn't want to transform all the text, such as "You're headed to." Would CSS toggling still be the preferred way?